sdk/python/tests/test_ci_workflow_contract.py

```
@@ -0,0 +1,247 @@
+        )
+
+
+@pytest.mark.parametrize("condition", ("false", "${{ matrix.run_job }}"))
+def test_job_view_rejects_disabled_or_conditional_jobs(condition: str) -> None:
+    workflow = f"""jobs:
+  python:
+    if: {condition}
+    steps:
+      - run: required-command
+"""
+    with pytest.raises(AssertionError, match="job python.*condition"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)